modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,62 +1,68 @@
 @import 'atlas-variables';
+@import 'cadmin-variables';
 
-.lfr-analytics-reports-panel {
-	&.open-admin-panel.sidenav-menu-slider {
-		visibility: visible;
-		width: 330px;
-	}
+html#{$cadmin-selector} {
+	.cadmin {
+		&.lfr-analytics-reports-panel {
+			&.open-admin-panel.sidenav-menu-slider {
+				visibility: visible;
+				width: 320px;
+			}
 
-	.sidebar {
-		border-left: 1px solid $gray-300;
-		box-shadow: none;
-	}
+			.sidebar {
+				border-left: 1px solid $gray-300;
+				box-shadow: none;
+			}
 
-	.sidebar-header {
-		height: 64px;
-		line-height: 32px;
+			.sidebar-header {
+				height: 64px;
+				line-height: 32px;
 
-		.lexicon-icon {
-			font-size: 14px;
-		}
-	}
+				.lexicon-icon {
+					font-size: 14px;
+				}
+			}
 
-	@media only screen and (min-width: 576px) {
-		.sidebar-header {
-			padding-bottom: 12px;
-			padding-top: 12px;
-		}
-	}
+			@media only screen and (min-width: 576px) {
+				.sidebar-header {
+					padding-bottom: 12px;
+					padding-top: 12px;
+				}
+			}
 
-	.sidebar-body {
-		bottom: 0;
-		left: 0;
-		position: absolute;
-		right: 0;
-		top: 64px;
-
-		.chart-loading-indicator {
-			position: absolute;
-			transform: translate(80%, 100%);
-			z-index: 1;
-		}
+			.sidebar-body {
+				bottom: 0;
+				left: 0;
+				padding: 0;
+				position: absolute;
+				right: 0;
+				top: 56px;
 
-		.loading-animation {
-			left: 0;
-			position: absolute;
-			right: 0;
-			top: 40%;
-		}
-	}
+				.chart-loading-indicator {
+					position: absolute;
+					transform: translate(80%, 100%);
+					z-index: 1;
+				}
 
-	.text-truncate-reverse {
-		direction: rtl;
-	}
+				.loading-animation {
+					left: 0;
+					position: absolute;
+					right: 0;
+					top: 40%;
+				}
+			}
 
-	.list-group {
-		margin-bottom: 0.5rem;
-	}
-}
+			.text-truncate-reverse {
+				direction: rtl;
+			}
 
-.dropdown-menu__translation {
-	max-height: 295px;
+			.list-group {
+				margin-bottom: 0.5rem;
```

Comment:
We need to convert `rem` values to `px`. The reason is the margin will increase or decrease if the `font-size` is modified at the document root (try on html element).  A `px` value will stay the same in that scenario.